demo5/listener.py

```diff
@@ -0,0 +1,38 @@
+import posix_ipc
+import selectors
+
+# This program uses `posix_ipc` together with the `selectors`library from the
+# Python standard library. `selectors` provides "high-level I/O multiplexing" akin to having an event library.
+
+# The message queue is created as usual
+mq = posix_ipc.MessageQueue("/python_ipc_test", flags=posix_ipc.O_CREAT)
+mq.block = False
+
+# Function is defined to handle events on the queue
+
+def accept(message_queue, mask):
+    (msg, prio) = message_queue.receive()
+    print(msg)
+    print(prio)
```

Comment:
Good idea to print the priority, it'd be nice to include some context (e.g. `"Message priority = {}"`)